docs/usage/msbuild-task.md

```diff
@@ -59,49 +84,59 @@ foreach (var field in fields)
 }
 ```
 
-**Specific variable**
+##### Specific variable
 
-```
+```c#
 var assemblyName = assembly.GetName().Name;
 var gitVersionInformationType = assembly.GetType(assemblyName + ".GitVersionInformation");
 var versionField = gitVersionInformationType.GetField("Major");
 Trace.WriteLine(versionField.GetValue(null));
 ```
 
-## Populate some MSBuild properties with version metadata
+### Populate some MSBuild properties with version metadata
 
-Task Name: `GitVersionTask.GetVersion`
+The sub-task `GitVersionTask.GetVersion` will write all the derived [variables](../more-info/variables.md)
+to MSBuild properties so the information can be used by other tooling in the build pipeline.
 
-At build time all the derived [variables](../more-info/variables.md) will be written to MSBuild properties so the information can be used by other tooling in the build pipeline.
+The class for `GitVersionTask.GetVersion` has a property for each variable. However at MSBuild time these
+properties a mapped to MSBuild properties that are prefixed with `GitVersion_`. This prevents conflicts with
```

Comment:
@asbjornu I realise this wasn't you, but can you change `these properties a mapped` to `these properties are mapped`